model/distro/db.go

```diff
@@ -74,6 +75,48 @@ func FindActive() ([]string, error) {
 	return out[0].Distros, nil
 }
 
+// FindActiveWithContainers gets all active distros that support containers
+func FindActiveWithContainers() ([]string, error) {
+	out := []struct {
+		Distros []string `bson:"distros"`
+	}{}
+	err := db.Aggregate(Collection, []bson.M{
```

Comment:
An aggregation isn't necessary here. I would do a simple find using the same query, and iterate through that extracting the IDs